messaging/src/main/java/org/jboss/as/messaging/jms/bridge/JMSBridgeService.java

```diff
@@ -93,18 +93,20 @@ public void run() {
     }
 
     public void startBridge() throws Exception {
-        if (moduleName == null) {
-            bridge.start();
+        final Module module;
+        if (moduleName != null) {
+            ModuleIdentifier moduleID = ModuleIdentifier.create(moduleName);
+            module = Module.getCallerModuleLoader().loadModule(moduleID);
```

Comment:
Why does this need getCallerModuleLoader? I realize the previous code was doing this, but  there are better ways to obtain a module loader. (getClallerModuleLoader use stack crawling which is quite slow)